app/enterprise/1.3-x/deployment/installation/amazon-linux-2.md

```diff
@@ -1,343 +1,320 @@
 
-The user and database can easily be created via the command line by using PostgreSQL’s command line tool `psql`
+    ```bash
+    $ sudo yum update -y
+    $ sudo yum install kong-enterprise-edition -y
+    ```    
 
-1. Change to the `postgres` user
-  ```
-  $ sudo -i -u postgres
-  ```
+### Copy the License File
+ 
+Copy the license file from your home directory to the `/etc/kong` directory like so:
 
-2. Launch the PostgreSQL command line tool `psql`
-  ```
-  $ psql
-  ```
+```bash
+$ sudo cp license.json /etc/kong/license.json
+```
 
-3. Create the `kong` user and database
-  ```
-  $ psql> CREATE USER kong; CREATE DATABASE kong OWNER kong; ALTER USER kong WITH password 'kong';
-  ```
-  > ⚠️Note: Make sure the username and password for the Kong Database are kept safe. We have used a simple example for illustration purposes only.
+## Step 3. Setup PostgreSQL
 
-4. Exit `psql`
-  ```
-  $ psql> \q
-  ```
+1. Install PostgreSQL.
 
-5. Return to the terminal
-  ```
-  $ exit
-  ```
-
-**iii. Connect Kong to the PostgreSQL database**
-
-Postgres uses `ident` authentication by default, to allow the `kong` user to communicate with the database locally we must change the authentication method to `md5` by modifying the Postgres configuration file. 
+    Follow the instructions avaialble at: https://www.postgresql.org/download/linux/redhat/ to install a supported version of PostgreSQL. Kong supports version 9.5 and higher. As an example, you may run a command set similar to:
 
+    ```bash
+    $ sudo amazon-linux-extras install postgresql9.6
+    $ sudo yum install postgresql postgresql-server
+    ```
 
-1. Open the configuration file:
-  ```
-  $ sudo vi /var/lib/pgsql/data/pg_hba.conf
-  ```
+2. Initialize the PostgreSQL database and enable automatic start.
 
-2. Change the `IPV4` local connection method to `md5`
-  ```
-    | TYPE | DATABASE | USER | ADDRESS      | METHOD |
-    | # IPv4 local connections:                      |
-    | host | all      | all  | 127.0.0.1/32 |  md5   |
-    | # IPv6 local connections:                      |
-    | host | all      | all  | ::1/128      | ident  |
-  ```
+    ```bash
+    $ sudo /usr/bin/postgresql-setup --initdb
+    $ sudo systemctl enable postgresql.service
+    $ sudo systemctl start postgresql
+    ```
+    
+3. Switch to PostgreSQL user and launch PostgreSQL.
 
-3. Save and exit the file, then restart Postgres
-  ```
-  $ sudo systemctl restart postgresql
-  ```
+    ```bash
+    $ sudo -i -u postgres
+    $ psql
+    ```
 
-Now that the database is set up, Kong must be given credentials to access the database. This is done by adding the Postgres user and password information to Kong’s configuration file.
+4. Create a Kong database with a username and password.
 
+  > ⚠️**Note**: Make sure the username and password for the Kong Database are
+  > kept safe. We have used a simple username and password for illustration purposes only. Note the database name, username and password for later. 
```

Comment:
Change "kept safe. We have used a simple username and password for illustration purposes only" to 
"kept safe. This example uses a simple username and password for illustration purposes only"